### PR TITLE
Adding Resource Management introduction

### PR DESCRIPTION
**Pull request by escowles**
Refs #356

### CODE REVIEW

**Comment by escowles** [comment]
I expect more wordsmithing here than in the main section 3 cleanup, so putting this in its own PR.

**Review by awoods — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -248,6 +248,23 @@ <h2>Conventions used in this document</h2>
...
+      <section id="resource-management-intro" class="informative">
+        <h2>Introduction</h2>
+        <p>
+          This section defines the core operations for managing content, using [[!LDP]] as a baseline. It
+          extends [[!LDP]] in several ways, including making some optional features mandatory and expanding
+          requirements related to binary content.
+        </p>
+        <p>
+          [[!LDP]] defines a hierarchy of resources, with a top-level distinction between RDF resources
+          (<a>LDP-RS</a>) and non-RDF resources (<a>LDP-NR</a>).  RDF resources are further divided into
```
Comment (zimeon) [inline index.html]: I think we should make it clear that not all RDF resources are containers. May also be easier to split into two sentence, something like:

> RDF resources may function as containers (<a>LDPC</a>) of other resources. There are several container subtypes, which are primarily differentiated by how much the container-member relationship can be configured.

Comment (escowles) [reply index.html]: 👍 just pushed an update to split into two sentences.

**Review by zimeon — DISMISSED** [review]

**Comment by zimeon** [comment]
Moving goalposts, no "Introduction" heading per #369

**Comment by escowles** [comment]
Updated to remove Introduction header and use blockquoted style instead.

**Review by dannylamb — DISMISSED** [review]
LGTM

**Review by awoods — APPROVED** [review]

**Review by dannylamb — APPROVED** [review]
Even better

**Review by zimeon — APPROVED** [review]

**Pull request merged**